ethkey/cli/src/main.rs

```diff
@@ -170,7 +170,7 @@ fn main() {
 fn display(keypair: KeyPair, mode: DisplayMode) -> String {
 	match mode {
 		DisplayMode::KeyPair => format!("{}", keypair),
-		DisplayMode::Secret => format!("{:?}", keypair.secret()),
+		DisplayMode::Secret => format!("{}", keypair.secret().to_hex()),
 		DisplayMode::Public => format!("{:?}", keypair.public()),
```

Comment:
I generally agree, but I believe it's not a scope of this pr. Here is why:

`Publish` and `Address` are just typenames for `H512` and `H160` whereas `Secret` is a wrapper around `H256`. Recently someone has changed the way `Secret` is displayed and broke tests. Our CI hasn't catched that, cause it was not running `ethkey-cli` tests... This change only makes our old tests pass...